blog/2024-08-14-conda-ecosystem-explained.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: "Conda Ecosystem Explained"
+slug: "2024-08-14-conda-ecosystem-explained"
+authors: [dashagurova, jezdez]
+tags: [conda, community, governance]
+description: |
+  Conda Ecosystem Explained
+---
+
+## What is the conda ecosystem?
+
+At its core, the conda ecosystem is a community supporting an open-source, language-agnostic, and multi-platform package management system.
+
+But what does a "package management ecosystem" really mean? Without diving too deep into every detail, here are the three main components conda community maintains:
+
+- **CLI Tools for Package Management**: conda command-line tools make it easy to install and manage dependencies in isolated development environments across Linux, Windows, and macOS. Conda works with packages written in any language, e.g. Python, R, C, or Fortran.
+- **Package Repository**: most publicly available conda packages are hosted on **anaconda.org**, organized into channels. Channels are like remote warehouses where packages are stored and can be pulled from. You can find channels maintained by community, companies, or individuals.
+- **Build Tools**: If you want to create and share your own conda packages, the ecosystem provides tools to help you build and distribute them efficiently.
+
+This system is bolstered by various other supporting projects, all of which you can explore on conda's [GitHub](https://github.com/conda).
+
+## Conda package & environment manager
+
+The cornerstone of this entire ecosystem is the [conda](https://github.com/conda/conda) package and environment manager. Originally developed by Anaconda, Inc. and implemented in Python, conda is not limited to Python \- it's a fully capable package manager that handles dependencies across multiple languages and platforms. Whether you're a data scientist working with Python, R, or a machine learning engineer dealing with libraries in C or Fortran, conda has you covered.
+
+Conda was open-source from the start and has grown significantly thanks to support from a broad community. Today, it's [governed by a multi-stakeholder steering council](https://github.com/conda/governance) under the fiscal sponsorship of the [NumFOCUS](https://numfocus.org) non-profit, ensuring it remains a balanced, community-driven project.
+
+And yes, conda is free. Always has been, always will be. This commitment to open-source has led to the development of several compatible alternatives within the conda ecosystem, like [Mamba](https://mamba.readthedocs.io/) — a faster reimplementation in C++ \- and [Pixi](https://pixi.sh/), a newer, Rust-based package management tool.
```

Comment:
```suggestion
And yes, `conda` is free. Always has been, always will be. This commitment to open-source has led to the development of several compatible alternatives within the conda ecosystem, like [Mamba](https://mamba.readthedocs.io/) — a faster reimplementation in C++ \- and [Pixi](https://pixi.sh/), a newer, Rust-based package management tool.
```

Reply:
I wonder if we should explain libmamba here. Someone reading this may take away that conda is slow, mamba is fast. However, today the differences are not the solver speed anymore.

Reply:
I concur, could you suggest an update?

Reply:
You marked as solved but if you still want a suggestion how about?

And yes, `conda` is free. Always has been, always will be. This commitment to open-source has led to the development of several compatible alternatives within the conda ecosystem, like [Mamba](https://mamba.readthedocs.io/) — a re-implementation in C++ \-, libmamba — a faster solver used by conda/mamba, and [Pixi](https://pixi.sh/), a newer, Rust-based package management tool.